src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -253,19 +254,19 @@ data class CommonCaseNoteFields(
       equalAmendments(this, other)
   }
 
-  private fun equalTypes(nomis: CommonCaseNoteFields): Boolean = (type == nomis.type || (type == "APP" && nomis.type == "CNOTE" && subType == "OUTCOME"))
+  private fun equalTypes(nomis: ComparisonCaseNote): Boolean = type == nomis.type || (type == "APP" && nomis.type == "CNOTE" && subType == "OUTCOME")
 
-  private fun equalUsers(other: CommonCaseNoteFields): Boolean {
-    val equal = other.nomisUsernames?.contains(dpsUsername) == true || dpsUsername == "OMS_OWNER"
+  private fun equalUsers(nomis: ComparisonCaseNote): Boolean {
```

Comment:
slightly confusing here, but the `equals` isn't symmetric as it always assumes DPS is on the left hand side of the comparison.  This duplicates the `equalTypes` method approach where the parameter name of `nomis` reflects that the nomis case note is on the right hand side